docs/xexchange-features/escrow.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 9
+id: xmex move
+---
+
+[comment]: # (mx-context-auto)
+
+# XMEX Move 
+ 
+XMEX is special for its transfer locking mechanism, since it is supposed to be bound to the account. But sometimes people have to move for various reasons, so do tokens.
+The XMEX Move function comes as a solution for this precise and usually inconvenient case.
+
+The XMEX Move (also known as Escrow) contract facilitates the transfer of XMEX from one account to another, with limitations in place to overcome scamming and abuse of token transfers that go against the idea of holding XMEX.
+Therefore, it shall be used mindfull of its strict conditions and, most likely, only in extreme situations.
+
+## XMEX Move flow
+
+ 1. Sender selects the batch of XMEX tokens to be transferred; after Move button clicked, sender has to fill in the receiver address and initiate the tokens transfer;
+ 2. Receiver will only be able to redeem the transferred token ONLY after a certain amount of days;
```

Comment:
Receiver will be able to redeem the transferred token ONLY after a certain amount of days;

Reply:
fixed